cardano-crypto-praos/cbits/vrf13_batchcompat/vrf_ietfdraft13.h

```diff
@@ -0,0 +1,10 @@
+#ifndef vrf_ietfdraft13_H
+#define vrf_ietfdraft13_H
+
+static const unsigned char SUITE = 0x04; /* ECVRF-ED25519-SHA512-ELL2 */
+
+static const unsigned char ZERO = 0x00;
+static const unsigned char TWO = 0x02;
+static const unsigned char THREE = 0x03;
```

Comment:
All of these constants are also defined in `vfr_ietfdraft03.h`. So maybe it would make sense to reuse them by defining in one header file. 

On the other hand if we do want to keep this duplicate definitions, then the current guard `#ifndef vrf_ietfdraft13_H` `#ifndef vrf_ietfdraft13_H` do not protect from `vfr_ietfdraft03.h` and `vfr_ietfdraft13.h` being included at the same time. So, it would probably make more sense to implement some sort of decent error saying that `ietfdraft13` and `ietfdraft03` cannot not be mixed, instead of `redefinition of 'ZERO'`  error.